techs/zetapack/factions/romans/units/guard_tower/guard_tower.xml

```diff
@@ -74,7 +74,7 @@
 			</sound>
 			<attack-strength value="135"/>
 			<attack-var value="25"/>
-			<attack-range value="12"/>
+			<attack-range value="15"/>
```

Comment:
This is fine by me, and I opened a ticket with an off-topic but related suggestion (#142)

Reply:
Again, 15 is a lot. I would take the tech tower as a reference. And then it could make sense for the range to be a little higher but not by much.

Reply:
Same reason for increase as persians. Actually, I was thinking about making the tech tower a cheaper, earlier unit since it is not as powerful as these towers. Maybe I have too many ideas buzzing xD